ohif-configurable/hp-extension/src/hp/hpCompare.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Hanging protocol which requires two studies, for which the first study and selected series
+ * has a series in the second study containing the same series description to show in the right hand panel.
+ */
+export default
+  {
+    name: 'Prior Compare',
+    hasUpdatedPriorsInformation: true,
+    numberOfPriorsReferenced: 1,
+    protocolMatchingRules: [
+      {
+        id: 'Two Studies',
+        weight: 1000,
+        attribute: 'StudyInstanceUID',
+        // The 'from' attribute says where to get the 'attribute' value from.  In this case
+        // prior means the second study in the study list.
```

Comment:
Does this rule accept a prior being two subsequent study items but not indexes 0,1?